readme.md

```diff
@@ -0,0 +1,237 @@
+# meta-xt-prod-devel
+
+## Content
+This document is long enough to has it's own content. So below you will find:
+* What is meta-xt-prod-devel
+* Quick start
+  * Some basic and most used commands
+* Usage and setup
+  * Serial over USB
+  * SSH over ethernet
+  * What to do "on board"
+* How to build
+  * Few build hints
+* How to install
+  * Write to microSD card
+  * Write to eMMC
+* How to prepare board (loaders and u-boot settings)
+* Supported boards
+* References
+
+Important note.
+All PC-related steps described bellow are intended to be run on Linux, and were tested on Ubuntu 18.
+Windows and MacOS are not supported, sorry.
+
+## What is meta-xt-prod-devel
+
+This is meta layer that allows you to run Linux+Linux or Linux+Android at same time under Xen hypervisor on some Renesas' R-CAR boards.
+List of fully or conditionaly supported boards can be found at the end of this document.
+Taking into account that you are interested in these sources, we suppose that you have some understanding what is Xen hypervisor.
+In case you need additional info look into last section of this document - "References".
+What prod-devel is?
+* it is some reference implementation that demonstrate simultanious run of few OS on same hardware
+Why it is under Xen hypervisor?
+* it can reboot promptly in case of some failure met
+* it can share hardware with some other special OS
```

Comment:
I don't entirely understand last statement

Reply:
Well, I see that this part is poorly written, but I do not have clear understanding how to do this properly.
So I rephrased this paragraph partially, but I'm open to put more meaningful text. Or anyone can push PR.